backend/stakepoold/signalsigterm.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2016 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+// +build darwin dragonfly freebsd linux netbsd openbsd solaris
```

Comment:
I asked you this question on Matrix, but reposting here for posterity.

> how did you identify that we need this? because I notice that has been newly added and there was nothing doing that before your PR

Reply:
Was a copy paste from wallet/dcrd and had not looked into its necessity. Since this is meant to run on linux, and we already had the working and tested signal.Notify(c, os.Interrupt) I will just use that. If this init turns out to be necessary, it can easily be added later.